docs/user-guide/admin-cli.md

```diff
@@ -351,6 +356,13 @@ Uuid                                  Name
 564d99a2-4097-9966-579f-3dc4082b10c9  photon7
 ```
 
+Note: If the VMs have volumes attached (containers running), their membership change i.e. changing the vmgroup to which
+they belong is not permitted. Make sure no volumes are attached.
+To do so:
+1. Get the list of containers running. (docker ps)
+2. If the container has any vDVS volume mounted (docker inspect container_name), stop the container.
+3. Ensure that the dvs volumes have status detached (docker volume inspect)
```

Comment:
nit: `dvs` => vDVS, let's stick with the same name to avoid confusion or may be just `vsphere driver` volumes to keep it simple.